docs/.eslintrc.json

```
@@ -0,0 +1,16 @@
+{
+	"extends": [
+		"wikimedia",
+		"wikimedia/node",
+		"plugin:json/recommended"
+	],
```

While `"wikimedia/node"` makes sense for our `tokens/` dir which only contains node.js code for building the tokens, the `docs/` and `vue-components/` folders are client code. I suggest something like the following eslint config:
```suggestion
	"extends": [
		"wikimedia",
		"wikimedia/client",
		"wikimedia/language/es6",
		"plugin:json/recommended"
	],
	"parserOptions": {
		"sourceType": "module"
	},
```